qiskit/visualization/pulse/qcstyle.py

```diff
@@ -17,48 +17,62 @@
 """
 Style sheets for pulse visualization.
 """
-from typing import Tuple, List
+from collections import namedtuple
+from typing import Tuple, Union, Optional
+
+import logging
+
+logger = logging.getLogger(__name__)
+ComplexColors = namedtuple('ComplexColors', ['real', 'imaginary'])
+SchedTableColors = namedtuple('SchedTableColors', ['time', 'channel', 'event'])
 
 
 class SchedStyle:
     """Style sheet for Qiskit-Pulse schedule drawer."""
-    def __init__(self, figsize: Tuple[float, float] = (10, 12),
+    def __init__(self, figsize: Optional[Tuple[float, float]] = (10.0, 12.0),
                  fig_unit_h_table: float = 0.4,
                  use_table: bool = True,
                  table_columns: int = 2,
                  table_font_size: int = 10,
                  axis_font_size: int = 18,
                  label_font_size: int = 10,
                  icon_font_size: int = 18,
-                 title_font_size: int = 25,
+                 title_font_size: Optional[Union[int, str]] = 25,
                  label_ch_linestyle: str = '--',
-                 label_ch_color: str = None,
+                 label_ch_color: str = '#222222',
                  label_ch_alpha: float = 0.3,
-                 d_ch_color: List[str] = None,
-                 u_ch_color: List[str] = None,
-                 m_ch_color: List[str] = None,
-                 s_ch_color: List[str] = None,
+                 d_ch_color: ComplexColors = ('#648fff', '#002999'),
+                 u_ch_color: ComplexColors = ('#ffb000', '#994A00'),
+                 m_ch_color: ComplexColors = ('#dc267f', '#760019'),
+                 s_ch_color: str = '#7da781',
                  s_ch_linestyle: str = '-',
-                 table_color: str = None,
-                 bg_color: str = None,
+                 table_color: Tuple[str, str, str] = ('#e0e0e0', '#f6f6f6', '#f6f6f6'),
+                 bg_color: str = '#f2f3f4',
                  num_points: int = 1000,
-                 dpi: int = 150,
+                 dpi: Optional[int] = 150,
                  remove_spacing: bool = True,
                  max_table_ratio: float = 0.5,
                  vertical_span: float = 0.2,
-                 axis_formatter: str = None):
+                 axis_formatter: str = '%s'):
         """Create new style sheet.
 
+        For any of the Optional fields, if that field is None then it will revert to its
+        matplotlib.rcParams counterpart. See for usage on rcParams.  Each argument that
+        is optonal also describes it's mapped rcParam key below.
+
         Args:
-            figsize: Size of figure.
+            figsize: Size of the figure in inches.
```

Comment:
the stylesheet is drawing backend agnostic, so we need to avoid mentioning rcParams here, i.e. the default parameter of drawing backend.

Reply:
What should i comment as the default parameter then?  It's not totally backend-agnostic at the moment if we take the `None` approach.  The default still relies on None.  If you want, I can just say it will default to the matplotlib defaults instead.

Reply:
Currently we are planning to introduce other drawing backends, as circuit drawer takes latex, matplotlib, and ASCII. The stylesheet should be common among those backends. So what I meant is just removing the explicit name of drawing backend.